client/components/aboutUs/HeroSection.js

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import styled from "styled-components";
+import { breakpoint } from "../../frontend-config";
+
+import { Heading1, TextBody } from "../shared/styles";
+
+const Section = styled.div`
+  text-align: center;
+  background: ${props => props.theme.colors.grey};
+  padding: 35px 75px 38px;
+  @media (min-width: ${breakpoint("lg")}) {
```

Comment:
I'd prefer spacing here.
It's becoming a standard to have spaces / linebreaks whenever you change the scope (ex. target new element or make media queries) 🙂